src/script/components/VerificationBadge/VerificationBadges.tsx

```diff
@@ -85,7 +116,10 @@ export const UserVerificationBadges = ({
   groupId?: string;
   isSelfUser?: boolean;
 }) => {
-  const {status: MLSStatus} = useUserIdentity(user.qualifiedId, groupId, isSelfUser);
+  const {deviceIdentities} = useUserIdentity(user.qualifiedId, groupId, isSelfUser);
+  const identity = deviceIdentities ? deviceIdentities[0] : undefined;
+  const {MLSStatus} = useMLSStatus(identity);
```

Comment:
you have access to the `User` at this point, why not giving it to `useMLSStatus` here and avoid one lookup later on?